src/serac/physics/state/state_manager.cpp

```diff
@@ -218,7 +218,7 @@ mfem::ParMesh* StateManager::setMesh(std::unique_ptr<mfem::ParMesh> pmesh, const
 
   // Functional needs the nodal grid function and neighbor data in the mesh
   auto& new_pmesh = mesh(mesh_tag);
-  new_pmesh.EnsureNodes();
+  new_pmesh.SetCurvature(1, false, -1, mfem::Ordering::byNODES);
```

Comment:
What is this `SetCurvature()` call doing, and what are the magic integers and boolean conveying?

Reply:
I'll add a comment about this method. Essentially, it does the same thing as `EnsureNodes`, but it is not hardcoded to use `mfem::Ordering::byVDIM`.